src/foam/nanos/http/FileServlet.java

```diff
@@ -48,8 +48,15 @@ private void fileNotFoundError(HttpServletResponse resp, String file) {
 
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
-    String pathInfo = req.getPathInfo();
-    String filePath = pathInfo.substring(SERVLET_NAME.length() + 2);
+    String pathInfo;
+    try {
+      pathInfo = req.getRequestURI();
+    } catch(Exception e) {
+      e.printStackTrace();
+      pathInfo = req.getPathInfo();
+    }
+
+    String filePath = pathInfo.substring(SERVLET_NAME.length() + (pathInfo.startsWith("/static/") ? 2 : 1));
```

Comment:
What is this check for? I could install multiple instances under multiple paths other than static.